iceoryx_posh/include/iceoryx_posh/internal/mepoo/chunk_management.hpp

```diff
@@ -36,20 +36,20 @@ struct alignas(32) ChunkManagement
     using referenceCounterBase_t = uint64_t;
     using referenceCounter_t = std::atomic<referenceCounterBase_t>;
 
-    ChunkManagement(const cxx::not_null<base_t*> f_chunkHeader,
-                    const cxx::not_null<MemPool*> f_mempool,
-                    const cxx::not_null<MemPool*> f_chunkManagementPool)
-        : m_chunkHeader(f_chunkHeader)
-        , m_mempool(f_mempool)
-        , m_chunkManagementPool(f_chunkManagementPool)
+    ChunkManagement(const cxx::not_null<base_t*> chunkHeader,
```

Comment:
I added ``noexcept`` but in this PR I would like to keep the signature since changing it will require a refactoring of the c'tors of ``RelativePointer`` and ``BaseRelativePointer``. This should be done in a follow-up PR.